app/code/Magento/Newsletter/Model/Subscriber.php

```diff
@@ -373,6 +386,28 @@ public function loadByCustomerId($customerId)
         return $this;
     }
 
+    /**
+     * Load subscriber info by customerId and storeId
+     *
+     * @param int $customerId
+     * @return $this
+     */
+    public function loadByCustomerIdAndStore($customerId)
```

Comment:
Unfortunately this class marked as @api, so according to https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/ we can't new public methods into it